app/js/controllers/foyer/individuForm.js

```diff
@@ -3,9 +3,9 @@
 angular.module('ddsApp').controller('FoyerIndividuFormCtrl', function($scope, individuRole, situationsFamiliales, SituationService, IndividuService) {
     $scope.statutsSpecifiques = IndividuService.getStatutsSpecifiques();
 
-    var options = {};
+    var options = $scope.options = {};
```

Comment:
If you do this, use only `$scope.options` everywhere. Otherwise, it feels we're manipulating a private variable while we're exposing everything to the view.